server/src/main/java/org/elasticsearch/index/mapper/RootObjectMapper.java

```diff
@@ -408,14 +409,16 @@ private static void validateDynamicTemplate(Mapper.TypeParser.ParserContext pars
 
         final boolean failInvalidDynamicTemplates = parserContext.indexVersionCreated().onOrAfter(Version.V_8_0_0);
         if (dynamicTemplateInvalid) {
-            String message = String.format(Locale.ROOT, "dynamic template [%s] has invalid content [%s]",
-                dynamicTemplate.getName(), Strings.toString(dynamicTemplate));
+            String format = "dynamic template [%s] has invalid content [%s], " +
+                "attempted to validate it with the following [match_mapping_type]: [%s]";
```

Comment:
Don't error messages usually use square brackets around values rather than names? In other words, should there be square brackets around `match_mapping_type`?

Reply:
You're right, I don't think so.